test/spec/modules/tcfControl_spec.js

```diff
@@ -452,16 +452,8 @@ describe('gdpr enforcement', function () {
       expectAllow(true, ufpdRule(activityParams(MODULE_TYPE_BIDDER, 'mockBidder')));
     });
 
-    it('should return deny when purpose 4 consent is withheld', () => {
-      setEnforcementConfig({
-        gdpr: {
-          rules: [{
-            purpose: 'personalizedAds',
-            enforcePurpose: true,
-            enforceVendor: true,
-          }]
-        }
-      });
+    it('should return deny by default when purpose 4 consent is withheld', () => {
+      setEnforcementConfig({});
```

Comment:
(GH wouldn't let me put a suggestion so I pushed this change) 

this makes more sense to me if the idea is to test the default behavior for p4; the 'someOtherPurpose` looked odd to me as it wasn't actually tested.